package.json

```diff
@@ -19,15 +19,16 @@
   "devDependencies": {
     "docco": "0.6.x",
     "eslint": "0.6.x",
-    "phantomjs": "~1.9.11",
-    "qunit-cli": "0.1.4",
+    "karma": "~0.12.31",
+    "karma-qunit": "~0.1.4",
+    "qunit-cli": "~0.1.4",
     "uglify-js": "2.4.x"
   },
   "scripts": {
-    "test": "npm run test-node && npm run test-browser && npm run lint",
-    "lint": "eslint underscore.js test/*.js test/vendor/runner.js",
+    "test": "npm run test-node && npm run lint",
+    "lint": "eslint underscore.js test/*.js",
     "test-node": "qunit-cli test/*.js",
-    "test-browser": "phantomjs test/vendor/runner.js test/index.html?noglobals=true",
+    "test-browser": "npm i karma-phantomjs-launcher && ./node_modules/karma/bin/karma start",
```

Comment:
npm won't install it every time, it will check compat. Also this avoids having to install phantomjs every time someone npm installs

And karma doesn't add its cli to bin